resources/user-agents/browsers/samsung/samsung-android-browser-3-0-to-6-0.json

```diff
@@ -215,19 +221,6 @@
             "Android_4_4"
           ]
         },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
-          "devices": {
-            "SM-G388F": "Samsung SM-G388F",
```

Comment:
I removed devices that weren't covered by tests, with the assumption that they don't support this version of the browser (2.0 and 2.1 were removed from this file, so perhaps these devices weren't upgraded to 3.0+ on this version of Android)